examples/lib/defaultHtmlStepUi.js

```diff
@@ -75,8 +75,20 @@ function DefaultHtmlStepUi(_sequencer, options) {
             '" value="' +
             paramVal +
             '" placeholder ="' +
-            (inputDesc.placeholder || "") +
-            '">';
+            (inputDesc.placeholder || "");
+            
+           if(inputDesc.type.toLowerCase() == "range")
```

Comment:
Oh wow, I love this implementation! What a great way to do it!!!! 🎉 @tech4GT check it out :-)

Reply:
Thanks 😸